src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -64,17 +72,26 @@ public async Task Run([BlobTrigger("sample-container/{name}", Connection = "Azur
                 {
                     var participantsEnumerator = csv.GetRecords<BssSubject>();
 
-                    await ProcessParticipantDataAsync(participantsEnumerator, participantUrl);
+                    await ProcessParticipantDataAsync(name,participantsEnumerator, participantUrl);
                 }
+
+                DateTime processingEnd = DateTime.UtcNow;
+
+                _logger.LogInformation("==================================================================\n"
+                                +"Participant Data: File {name} processed successfully.\n"
+                                +"Start Time: {processingStart}, End Time: {processingEnd}.\n"
+                                +"Rows Processed: {participantRowIndex}, Success: {successCount}, Failures: {failureCount}"
+                                ,name,processingStart,processingEnd,participantRowIndex,participantSuccessCount, participantFailureCount );
             }
             else
             {
                 _logger.LogError("fileName is invalid. file name: {Name}", name);
                 return;
             }
 
-            _logger.LogInformation("Data processed successfully.");
+            //_logger.LogInformation("Data processed successfully.");
```

Comment:
Do we need this comnent here?

Reply:
not actually. kept it as a landmark. will be removed.